### DISCUSSION TITLE
Usage of fixprotocol.io for Orchestra

### DISCUSSION THREAD

**Discussion by kleihan** [FIXTradingCommunity/fix-orchestra#209] [Ideas]
https://fixprotocol.io is a website for the GitHub presence of the FIX Trading Community. It is maintained through a public GitHub project at https://github.com/FIXTradingCommunity/fixtradingcommunity.github.io/. It has not been advertised and currently does not provide access to its contents, only links to the FIX website and FIXimate.

However, it contains the final versions of the repository and interfaces schemas for Orchestra v1.0 as well as the schema for SBE 1.0. These are accessible by directly using the URLs contained in the Orchestra spec, e.g. http://fixprotocol.io/2020/orchestra/repository.

The FIX website currently has a landing page for FIX Technical Standards () with a link to the online version of the Orchestra spec. These contain URLs to the schemas on fixprotoocol.io.

### Questions for discussion:
- Should we continue to use fixprotocol.io as landing page for FIX Trading Community on GitHub?
- If so, what should be accessible from that page?
- Should we add schemas of published Orchestra RCs to the GitHub project so that URLs like http://fixprotocol.io/2023/orchestra/repository can be resolved and provide access to the latest schema?

**Comment by patricklucas** [comment]
Having fixprotocol.io be resolvable and, in general, having XML namespace URIs actually lead to a relevant page, is valuable in my opinion.

Keeping the site thin, and primarily linking out to other FIX sites and resources also seems preferable to duplicating content found elsewhere—the mission being to serve to close the loop with namespace URIs more than be a resource on its own.

I'm not sure that this site needs to actually host the schema files, serving as an additional source of truth to the fix-orchestra-spec repo. What if we instead changed fixprotocol.io to still have a landing page for each published standard's namespace URI(s), but then have those pages link to the appropriate subdirectory of the fix-orchestra-spec repo on GitHub?

We could also add a lightweight templating/build process that would generate the appropriate directories/index.html files for each published spec (including RCs), for instance by cloning the fix-orchestra-spec repo at build time, and discovering the schemas and their namespace URIs.
